test/fixtures/simple_regional_with_networking/variables.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The project ID to host the cluster in"
+}
+
+variable "region" {
+  description = "The region to host the cluster in"
+  default     = "us-east4"
+}
+
+variable "network" {
+  description = "The VPC network created to host the cluster in"
+  default     = "gke-network"
```

Comment:
Since the example itself has defaults, no need to set these as variables in the fixture. Instead just omit them from the fixture entirely and let the example defaults be used.

Reply:
@morgante quick question about this. I wanted to do it that way, but how do I reference the output for the tests? i.e how do I reference the output of the underlying VPC module from example? Currently I seem to be only getting the outputs of GKE module as the output of example.

Reply:
You need to add it as an output in the example.

Reply:
Thanks, done